docs/user-guide/compute-resources.md

```diff
@@ -269,6 +269,86 @@ LastState: map[terminated:map[exitCode:137 reason:OOM Killed startedAt:2015-07-0
 
 We can see that this container was terminated because `reason:OOM Killed`, where *OOM* stands for Out Of Memory.
 
+## Alpha Features
```

Comment:
Current master looks the same when rendered with vanilla Github markdown preview (see https://github.com/kubernetes/kubernetes.github.io/blob/7112d4c53d8a6e34f3a43c8f4449403617f091e7/docs/user-guide/compute-resources.md#my-container-is-terminated)